packages/node_modules/@webex/http-core/src/request/request.shim.js

```diff
@@ -42,6 +45,12 @@ export default function _request(options) {
 
       /* istanbul ignore else */
       if (response) {
+        if (response.statusCode >= 400) {
```

Comment:
We might need to add something here for response.statusCode <= 0

Reply:
I don't think it will be less than zero but it could be zero

Reply:
This should not be an issue here.  The changing it to zero occurs a few lines below this code in the else:
      else {
        resolve({
          statusCode: 0,
          options,
          headers: options.headers,
          method: options.method,
          url: options.uri,
          body: error
        });
      }
In that case, hopefully we were able to log the error with the warning above this:
      if (error) {
        options.logger.warn(error);
      }
But if we didn't log it there (ie error is not truthy), the thing is, there's really nothing to log, there was no response to log (this if block is checking the existence of response).  My goal here was to log the network level response.